src/Taxonomy.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Taxonomy class file.
+ *
+ * @package  studiometa/wp-factories
+ */
+
+namespace Studiometa\WP\Factories;
+
+/**
+ * Taxonomy Class
+ */
+class Taxonomy {
+	/**
+	 * Generate labels
+	 *
+	 * @param  string $singular           Singular name.
+	 * @param  string $plural             Plural name.
+	 * @param  string $translation_domain Translation domain name.
+	 * @return array                      Labels
+	 */
+	public static function generate_labels( string $singular, string $plural, string $translation_domain = null ) {
+		$singular_lowercase = strtolower( $singular );
```

Comment:
Same as for the CPT class, I think the `$singular_lowercase` and the `$plural_lowercase` should be arguments with fallbacks to `strtolower( ... )`.